test/e2e/fixture-builder.js

```diff
@@ -1819,6 +1824,141 @@ class FixtureBuilder {
     return this.withNameController({ names: {} });
   }
 
+  withTrezorAccount() {
```

Comment:
since importing the trezor account has already been tested in the `trezor-account` spec, here and in the subsequent trezor tests, we can use fixtures and skip those steps directly. Notice the keyring type is must be `Trezor Hardware`, otherwise we would be testing a regular imported account.